src/Equations/Fluid/KineticEquationTermIntegratedOverMomentum.cpp

```diff
@@ -30,6 +30,10 @@ KineticEquationTermIntegratedOverMomentum::KineticEquationTermIntegratedOverMome
     rebuildOperator(rebuild), scaleFactor(scaleFactor) {
     
     SetName("KineticEquationTermIntegratedOverMomentum");
+    
+    this->CsetJacobian = new Mat[u->GetNUnknowns()];
```

Comment:
This should also be deleted in the destructor, right?

Reply:
Isn't it already (line 48-49)?

Reply:
No, I meant deleting the pointer:
```c++
delete [] this->CsetJacobian;
```

Reply:
What is on lines 48-49 now is also needed, but it doesn't undo the call to ``new`` on line 34.

Reply:
Ah, right, I updated it now!